examples/lang/erc20/src/lib.rs

```diff
@@ -168,10 +166,10 @@ mod tests {
 
     #[test]
     fn transfer_works() {
-        let alice = AccountId::try_from([0x0; 32]).unwrap();
-        let bob = AccountId::try_from([0x1; 32]).unwrap();
+        let alice: AccountId = From::from([0x0; 32]);
```

Comment:
Wouldn't it be more readable to just write: `let alice = AccountId::from([0x0; 32]);`
Open to opinions.

Reply:
Yes I agree. This is a relic of an earlier iteration